arxiv/how_can_we_be_so_dense/README.md

```diff
@@ -30,6 +30,14 @@ docker build -t htmpaper .
 docker run -it htmpaper /bin/bash
 ```  
 
+After running this last command, you'll be given a shell into the docker container. To get into the directory with this README file:
+
+```
+cd arxiv/how_can_we_be_so_dense
```

Comment:
This is not correct. The docker home directory should already be in this directory